Makefile

```diff
@@ -108,15 +116,17 @@ help: ## to get help
 	awk 'BEGIN {FS = ":.*?## "}; {printf "make ${CYAN}%-30s${CNone} %s\n", $$1, $$2}'
 
 .PHONY: lint
-lint: go-lint go-mod-check docker-lint ## to lint all
```

Comment:
Could you explain why we are removing this?

Reply:
`go-mod-check` is not linting, is it? 
If it is required , it can be triggered indepentanly with make go-mod-check. 

Or maybe create a new target named make checks/tests to include that process.

Reply:
Not linting but something we'd probably want to check as part of CI. So happy to move it out of here but would prefer to retain it.

Reply:
Noted. Will create a seperate target for these kind of checks